cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaCluster.java

```diff
@@ -1088,6 +1088,10 @@ private Map<String, String> preparePodSetAnnotations(Storage storage)   {
         controllerAnnotations.put(ANNO_STRIMZI_IO_KAFKA_VERSION, kafkaVersion.version());
         controllerAnnotations.put(Annotations.ANNO_STRIMZI_IO_STORAGE, ModelUtils.encodeStorageToJson(storage));
 
+        //Take ownership of the rolling update annotation (for SSA) and reset it
+        //as (potential) one-time roll was already ran by reconcile loop run if it was needed.
+        controllerAnnotations.put(Annotations.ANNO_STRIMZI_IO_MANUAL_ROLLING_UPDATE, "false");
+
```

Comment:
We first need to take control of it by setting any value and then setting null would you remove the annotation.
Since we are not owners of the annotation setting null (so not including annotation in the request) would be just noop.

Reply:
Ok. It is ugly, but I guess having it set to false is better than having some complicated logic to set it first to false and then to null.